connect/cli/plugins/project/extension_helpers.py

```diff
@@ -93,9 +96,52 @@ def validate_extension_project(project_dir: str):
     extension_dict = _project_descriptor_validations(project_dir)
     _entrypoint_validations(project_dir, extension_dict)
 
+    _runner_version_validation(project_dir)
+
     click.secho(f'Extension Project {project_dir} has been successfully validated.', fg='green')
 
 
+def _runner_version_validation(project_dir):
+    res = requests.get(PYPI_EXTENSION_RUNNER_URL)
+    if res.status_code == 200:
+        latest_version = res.json()['info']['version']
```

Comment:
i will feel more confortable that you ensure that is really a json......i saw it tons of times that systems returns valid code but not what is expected (for example under maintenance situations)

Reply:
Done!!!
Now the data retrieved is checked...just in case ;-)